docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -6,7 +6,7 @@
 = Migrating databases to MariaDB instances
 
 This document describes how to move the databases from the original
-{osp_prev_long} ({OpenStackShort}) deployment to the MariaDB instances in the {OpenShift} cluster.
+{rhos_prev_long} ({OpenStackShort}) deployment to the MariaDB instances in the {OpenShift} cluster.
```

Comment:
Is OpenStackShort attribute in another file other than rhoso_attributes?

Reply:
Yes, it's in the adoption-attributes file.